client/src/components/styledComponents.js

```diff
@@ -0,0 +1,115 @@
+import { Input, Item, Card, Image } from 'semantic-ui-react';
+
+import styled from 'styled-components';
+
+//App
+
+export const AppPage = styled.div`
+  font-size: 20px;
+  font-family: Verdana;
+  margin: auto;
+  padding-top: 30px;
+  width: 80%;
+  @media screen and (max-width: 850px) {
+    width: 95%;
+  }
+`;
+
+//SearchBar
+
+export const SearchInput = styled(Input)`
+  margin-bottom: 35px;
+`;
+export const Icon = styled.i`
+  background-color: rgb(255, 65, 90);
+`;
+
+// FilmDetails
+
+export const StyledItem = styled(Item)`
+  text-align: left;
+`;
+export const CharactersButton = styled(Item.Header)`
+  font-size: 24px;
+  margin: 12px;
+  cursor: pointer;
+`;
+
+export const CharactersList = styled(Card.Group)`
+  @media screen and (max-width: 700px) {
+    margin-top: 10px !important;
+  }
+`;
+export const MovieInfoText = styled.p`
+  font-size: 24px;
+  @media screen and (max-width: 700px) {
+    font-size: 18px;
+  }
+`;
+
+export const PageContainer = styled.div`
+  width: 100%;
+  height: auto;
```

Comment:
```suggestion
  height: 100vh;
```

This prevents the text being readjusted and shifting horizontally when expanding the character list.